docs/astro/src/content/docs/guide/platforms/mobile/general.mdx

```diff
@@ -0,0 +1,181 @@
+touch interfaces are expected to scroll while panning the whole view.
+
+This can be enabled in Slint like this:
+
+```slint
+ScrollView {
+    mouse-drag-pan-enabled: true;
+
+    // Your own elements here
+}
+```
+
+Note that this also enables the behavior when a mouse is attached, which might not be desired. This will be fixed in
+a future version of Slint (with the goal that no special setting will be necessary in the first place).
+
+## Safe Area
+
+Operating system developers for mobile devices try to improve the issue of the small screen by reducing the amount of
+system-level overlays to a bare minimum. These days, this means that applications usually have access to the entire screen,
+but there are a few system items overlaid, for example the cell tower reception and battery status. On Android, this also
+includes navigation buttons. Also, many cellphone devices embed the front-facing camera into the display, leaving only a
+small non-rectangular area inaccessible ("the notch").
+
+All of these overlays result in certain areas of the screen being accessible by the application for drawing (except that
+there are icons drawn on top), but not allowing user interaction. For example, when buttons are located there, users can
+not tap them. So, the concept of a "safe area" was introduced. That is the area where an application can expect users to
+be able to interact with interface elements. It is defined by an inset, meaning that it defines the thickness of an
+invisible border inside the application's window which should only contain background elements (like background images,
+patterns, color). This safe area can also change in size at runtime, for example on Android when the navigation buttons
+at the bottom are shown and hidden.
+
+In Slint, this area is exposed with four properties on the Window element:
+
+* `safe-area-inset-top`
+* `safe-area-inset-bottom`
+* `safe-area-inset-left`
+* `safe-area-inset-right`
+
+They default to 0 and stay on that value on other platforms. Usually, mobile platforms only set top and bottom to greater
+than zero, but this is not guaranteed.
+
+If you want to place a rectangle to visualize the safe area (for example for debugging), you can do it like this:
+
+```slint
+export component MainWindow inherits Window {
+    Rectangle {
+        background: yellow;
+        x: root.safe-area-inset-left;
+        y: root.safe-area-inset-top;
+        width: root.width - root.safe-area-inset-right - root.safe-area-inset-left;
+        height: root.height - root.safe-area-inset-bottom - root.safe-area-inset-top;
+    }
+}
+```
+
+## Keyboard Handling
+
+Modern smartphones since the introduction of the iPhone usually don't feature a hardware keyboard for text input. There are
+accessories like Bluetooth or USB keyboards and special phone cases that do provide such a feature, but there always has to
+be a software fallback, which we call the "virtual keyboard".
```

Comment:
```suggestion
Since the introduction of the iPhone, modern smartphones usually don't feature a hardware keyboard for text input. There are
accessories like Bluetooth or USB keyboards and special phone cases that do provide such a feature, but there always has to
be a software fallback, the "virtual keyboard".
```